api/bibliotheca.py

```diff
@@ -894,23 +909,7 @@ def date_from_subtag(self, tag, key, required=True):
         return self.parse_date(value)
 
 
-class BibliothecaException(Exception):
```

Comment:
Replaced this exception with `RemoteInitiatedServerError`, so that we get problem details from all the exceptions thrown by the Bibliotecha API.